packages/calypso-products/src/types.ts

```diff
@@ -192,6 +194,13 @@ export type FeatureGroup = {
 };
 export type FeatureGroupMap = Record< FeatureGroupSlug, FeatureGroup >;
 
+export type StorageOption = {
+	slug: string;
+	// Determines if the storage option is an add-on that can be purchased. There are a mixture of patterns
+	// to identify add-ons for now, and we're temporarily adding one more
+	isAddOn: boolean;
```

Comment:
could this be set to optional and avoid all the `isAddOn: false` assertions above (that make up most cases)

Reply:
Yes! I'll make a note of this and take care of it in either an upcoming or separate PR